node/cli/src/command.rs

```diff
@@ -73,54 +73,55 @@ fn load_spec(id: &str) -> std::result::Result<Box<dyn ChainSpec>, String> {
 		"frequency" => return Ok(Box::new(chain_spec::frequency::load_frequency_spec())),
 		#[cfg(feature = "frequency-no-relay")]
 		"dev" | "frequency-no-relay" =>
-			return Ok(Box::new(chain_spec::frequency_rococo::development_config())),
+			return Ok(Box::new(chain_spec::frequency_dev::development_config())),
 		#[cfg(feature = "frequency-local")]
 		"frequency-rococo-local" =>
 			return Ok(Box::new(chain_spec::frequency_rococo::local_rococo_testnet_config())),
 		#[cfg(feature = "frequency-local")]
 		"frequency-paseo-local" =>
 			return Ok(Box::new(chain_spec::frequency_paseo::local_paseo_testnet_config())),
 		#[cfg(feature = "frequency-testnet")]
-		"frequency-paseo" => return Ok(Box::new(chain_spec::frequency_paseo::frequency_paseo_testnet())),
+		"frequency-testnet" | "frequency-paseo" | "paseo" | "testnet" =>
+			return Ok(Box::new(chain_spec::frequency_paseo::load_frequency_paseo_spec())),
 		#[cfg(feature = "frequency-testnet")]
-		"frequency-testnet" | "frequency-rococo" | "rococo" | "testnet" =>
+		"frequency-rococo" | "rococo" =>
 			return Ok(Box::new(chain_spec::frequency_rococo::load_frequency_rococo_spec())),
 		path => {
 			if path.is_empty() {
 				if cfg!(feature = "frequency") {
 					#[cfg(feature = "frequency")]
 					{
-						return Ok(Box::new(chain_spec::frequency::load_frequency_spec()))
+						return Ok(Box::new(chain_spec::frequency::load_frequency_spec()));
 					}
 					#[cfg(not(feature = "frequency"))]
-					return Err("Frequency runtime is not available.".into())
+					return Err("Frequency runtime is not available.".into());
 				} else if cfg!(feature = "frequency-no-relay") {
 					#[cfg(feature = "frequency-no-relay")]
 					{
-						return Ok(Box::new(chain_spec::frequency_rococo::development_config()))
+						return Ok(Box::new(chain_spec::frequency_dev::development_config()));
```

Comment:
New `frequency_dev` chain spec section to handle the dev instead of lumping it in with rococo